.travis.yml

```diff
@@ -39,11 +39,15 @@ script:
     [ -z "$lint" -o "$lint" = "Lint-free!" ]
   - go vet $(go list ./... 2>/dev/null | grep -v /vendor/)
   - go install github.com/keybase/kbfs/...
-  - cd libkbfs && go test -i && go test -race -c && ./libkbfs.test -test.timeout 4m
+  - cd libkbfs && go test -i && go test -race -c && ./libkbfs.test -test.timeout 1m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./libkbfs.test -test.timeout 1m
   - cd ../libfuse && go test -i && go test -c && ./libfuse.test -test.timeout 2m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./libfuse.test -test.timeout 2m
   - cd ../test
   - go test -i -tags fuse && go test -race -c && ./test.test -test.timeout 4m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./test.test -test.timeout 4m
   - go test -c -tags fuse && ./test.test -test.timeout 4m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir go test -c -tags fuse && ./test.test -test.timeout 4m
```

Comment:
As discussed, do only in-memory bserver for libkbfs, and on-disk bserver for everything else.